pkg/cli/kubernetes/logstream/stern.go

```diff
@@ -99,7 +99,10 @@ func (i *Impl) Stream(ctx context.Context, options Options) error {
 	// This will block until the context is cancelled.
 	err = stern.Run(ctx, options.KubeClient, &cfg)
 	if err != nil {
-		return err
+		// Not returning the error and just logging is intentional!
+		// We don't want the process to exit if there's an error streaming logs.
+		// We just want to log the error and continue.
+		fmt.Println("Error running stern: ", err)
```

Comment:
Do they provide a reliable error code that we can use to filter out and ignore only the limit exceeded error?

Reply:
From what I see, they don't. I can check one more time and let you know.